sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="2003-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="GNU GPL v2"
```

Comment:
It looks like this would be more appropriate.
```diff
-LICENSE="GNU GPL v2"
+LICENSE="MIT
+	GNU GPL v2"
```

Reply:
Or the other way around, perhaps...

Reply:
The main library (`libatomic_ops`) is MIT licensed, so I would prefer having MIT as the first entry.

Also, don't forget this little [custom license][0]

[0]: https://github.com/ivmai/libatomic_ops/blob/v7.6.2/doc/LICENSING.txt#L43